Cargo.toml

```diff
@@ -256,4 +256,4 @@ lto = true
 
 [profile.profiling]
 inherits = "release"
-debug = "line-tables-only"
+debug = "limited"
```

Comment:
so you've changed to `limited` now?

Reply:
Yes, i've compared profiles with `lines-tables-only` and `limited`.
And with `lines-tables-only` resulting profile looks very confusing because instead of `sumeragi::main_loop::run` funcion name would be just `run` which make it very confusing. 
Looks like i didn't noticed it before.